dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -317,18 +293,40 @@ export const Card = ({
 									</AvatarContainer>
 								</Hide>
 							)}
-							{/* Show the card footer in the same column as the headline content */}
-							{!moreThanTwoSubLinks ? (
-								renderFooter({ forceVertical: true })
-							) : (
-								<></>
-							)}
-						</div>
-					</ContentWrapper>
-				</CardLayout>
-				{/* If there are more than two sublinks break footer out of the headline column into a row below */}
-				{moreThanTwoSubLinks ? renderFooter({}) : <></>}
-			</CardWrapper>
-		</CardLink>
+						</>
+					</Flex>
+					<div>
+						{trailText && (
```

Comment:
Could be wrong but I think the trailtext should be hidden when there are nested links... I'm not sure if this is decided by the layout or if its the same for all cards regardless of layout.

Reply:
You do sometimes get both together
<img width="710" alt="Screenshot 2022-05-03 at 09 50 15" src="https://user-images.githubusercontent.com/1336821/166426997-1a01687f-39af-4526-b80f-9fa0e4102b5d.png">
I don't have a complete picture for the rules yet but they do seem to be layout specific. Indeed it looks like specific cards in certain positions have their own rules about what appears in them

Reply:
Ach, so complicated!